shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -317,6 +318,29 @@ - (NSOpenGLContext*)resourceContext {
   return _resourceContext;
 }
 
+- (float)displayRefreshRate {
+  NSDictionary* description = [[NSScreen mainScreen] deviceDescription];
+  CGDirectDisplayID displayId = [[description objectForKey:@"NSScreenNumber"] unsignedIntValue];
+  CGDisplayModeRef displayModeRef = CGDisplayCopyDisplayMode(displayId);
+  double refreshRate = CGDisplayModeGetRefreshRate(displayModeRef);
+
+  // The above can return zero always on some devices.
+  // See: https://github.com/glfw/glfw/issues/137
```

Comment:
Why not link to [the actual documentation](https://developer.apple.com/documentation/coregraphics/1454661-cgdisplaymodegetrefreshrate) for this, rather than a comment referencing the documentation?

Reply:
Agreed, will do.